entrypoint

```diff
@@ -2,7 +2,7 @@
 
 set -e
 
-RUN_COMMAND="talisker.gunicorn.gevent webapp.app:app --bind $1 --worker-class gevent --name talisker-`hostname`"
+RUN_COMMAND="talisker.gunicorn webapp.app:app --bind $1 --worker-class sync --workers 4 --name talisker-`hostname`"
```

Comment:
Why 4? I think we now use 8 in some places, 5 in others, and here 4. Did you copy this from somewhere?